README.md

```diff
@@ -20,27 +16,29 @@ $ npm run serve
 ```
 
 # Cleanup
-Stop the container
+Stop the vm without deleting the database
+```
+$ docker-compose stop
+```
+
+Stop the vm and remove all data
 ```
-$ docker stop pg-docker
+$ docker-compose down
 ```
 
 # Connect to psql
 ```
-$ psql -h localhost -U postgres -d postgres
+$ psql -h localhost -d postgres
 ```
-The password should be `docker`.
 
 # Create a dump of the schema
 ```
-$ docker exec pg-docker pg_dump -s -U postgres user_auth > schema.sql
+$ docker exec user_auth_postgres_1 pg_dump -s -U $USER user_auth > schema.sql
```

Comment:
You're exposing the ports, you don't need docker commands, and you don't need the user anymore:
```sh
pg_dump -s --clean --no-owner -h localhost user_auth > schema.sql
```